build_rules.gradle

```diff
@@ -508,18 +508,26 @@ ext.applyAvroNature = {
   apply plugin: "com.commercehub.gradle.plugin.avro"
 }
 
+// A class defining the set of configurable properties for createJavaQuickstartValidationTask
+class JavaQuickstartConfiguration {
+  String name
+  String gcpProject = ''
```

Comment:
Please add comments for the purpose of the different fields.

Reply:
Done.